DER/DerivativesContracts/Options.rdf

```diff
@@ -594,23 +469,40 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">An exercise schedule may be as simple as a single date or date period. However, in more complex cases, it may be an ad hoc schedule of individual dates, or a regular schedule of periodic exercise dates.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
unclear how this would apply to a vanilla option that a) expires and b) is exercised prior to expiry. Does this represent the settlement period e.g. T+2 days?

Reply:
@rivettp I think settlement would be something separate, not this.  The schedule would be the expiry date, even if the option might be exercised prior to that date.